packages/orbit-components/src/InputField/index.js

```diff
@@ -434,7 +435,7 @@ const InputField: InputFieldType = React.forwardRef((props, ref) => {
             prefix && <Prefix size={size}>{prefix}</Prefix>
           )}
           {label && inlineLabel && (
-            <StyledInlineLabel ref={labelRef} size={size}>
+            <StyledInlineLabel isErrorForms={!tags && (error || help)} ref={labelRef} size={size}>
```

Comment:
"Error forms" is an internal name of the feature that doesn't really describe what it is, instead I recommend adding two boolean props that have a clear meaning and derive styles from those.
```suggestion
            <StyledInlineLabel hasTags={!!tags} hasFeedback={!!(error || help)} ref={labelRef} size={size}>
```